.github/workflows/release.yml

```
@@ -282,22 +282,22 @@ jobs:
-          path: apps/flipper/build/windows/x64/runner/Release/flipper_rw.msix
+      - name: Extract msix_version
+        id: get_version
+        shell: powershell
+        run: |
+          $MSIX_VERSION = (Get-Content -Path "apps/flipper/pubspec.yaml" -Raw) -match 'msix_config:\s*([\s\S]*?)\bmsix_version:\s*(\d+\.\d+\.\d+\.\d+)\b' | ForEach-Object { if ($matches.Count -ge 2) { $matches[2] } else { Write-Output "No msix_version found"; exit 1 } }; echo "::set-output name=version::$MSIX_VERSION"
 
-      - name: Create Release
-        uses: softprops/action-gh-release@v1  # Or equivalent release action
-        # if: startsWith(github.ref, 'refs/tags/')  # Only trigger on tag pushes (releases)
+      - name: Create Debug Releasse
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)